h5py/_hl/files.py

```diff
@@ -404,12 +416,24 @@ def __init__(self, name, mode='r', driver=None,
             The smallest free-space section size that the free space manager
             will track.  Only allowed when creating a new file.  The default
             value is 1.
+        page_buf_size
+            Page buffer size in bytes. Only allowed in "r" or "r+" modes and for
+            HDF5 files created with fs_strategy="page". Must be a power of two
+            value and greater or equal than the file space page size when
+            creating the file. It is not used by default.
```

Comment:
This seems like fairly specific conditions. If you use a value smaller than the file space page size, or the file wasn't created with `fs_strategy="page"`, does HDF5 throw an error, or does it just quietly not use the page buffer?

Reply:
(Just found in the HDF5 docs that it will error)